firebase-inappmessaging-display/src/main/java/com/google/firebase/inappmessaging/display/FirebaseInAppMessagingDisplay.java

```diff
@@ -82,6 +82,8 @@ public class FirebaseInAppMessagingDisplay extends FirebaseInAppMessagingDisplay
       20 * 1000; // auto dismiss after 20 seconds for banner
   static final long INTERVAL_MILLIS = 1000;
 
+  public boolean useCustomTabs = true;
```

Comment:
We should probably make this variable private and have public getters/setters to be consistent with other Firebase APIs.